### ISSUE TITLE
Dropbox response: "This app has reached its user limit"

### ISSUE THREAD

**Issue by gwarah** [syncany/syncany#576]
Hi, recently I've tried to add a dropbox folder and got this message:

"This app has reached its user limit. Contact the app developer and ask them to use the Dropbox API App Console to increase their app's user limit."

Well, so I've tried to create a new local folder repository on dropbox directory: something like [dropbox-pah]\syncany

In other pc of mine, could I read these repository by syncany accessing local folder too?

**Comment by internetimagery** [comment]
Yeah it seems the app key has too many users. Unsure why they don't update it or allow users to add their own keys.

You can in the meantime make a folder in your dropbox folder. Then use the "local storage" option to put the files in there and sync it up that way. The team here has done a great job of deduplicating the files so the result is often smaller than you would otherwise have in your dropbox if you put in the files manually. :)

**Comment by gwarah** [comment]
Ok, tks @internetimagery by the way, any project to syncany could have google drive repository?
